Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,628 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
+| Costa Rica                       | North America |
+| Cuba                             | North America |
+| Cayman Islands                   | North America |
+| Dominica                         | North America |
+| Dominican Republic               | North America |
+| Guadeloupe                       | North America |
+| Grenada                          | North America |
+| Greenland                        | North America |
+| Guatemala                        | North America |
+| Honduras                         | North America |
+| Haiti                            | North America |
+| Jamaica                          | North America |
+| Saint Kitts and Nevis            | North America |
+| Saint Lucia                      | North America |
+| Mexico                           | North America |
+| Montserrat                       | North America |
+| Martinique                       | North America |
+| Nicaragua                        | North America |
+| Panama                           | North America |
+| Puerto Rico                      | North America |
+| El Salvador                      | North America |
+| Saint Pierre and Miquelon        | North America |
+| Turks and Caicos Islands         | North America |
+| Trinidad and Tobago              | North America |
+| United States                    | North America |
+| Saint Vincent and the Grenadines | North America |
+| Virgin Islands, British          | North America |
+| Virgin Islands, U.S.             | North America |
++----------------------------------+---------------+
+37 rows in set (0,00 sec)
+
+
+4
+
+mysql> select SurfaceArea,Name from country ORDER BY SurfaceArea  DESC LIMIT 10 ;
++-------------+--------------------+
+| SurfaceArea | Name               |
++-------------+--------------------+
+| 17075400.00 | Russian Federation |
+| 13120000.00 | Antarctica         |
+|  9970610.00 | Canada             |
+|  9572900.00 | China              |
+|  9363520.00 | United States      |
+|  8547403.00 | Brazil             |
+|  7741220.00 | Australia          |
+|  3287263.00 | India              |
+|  2780400.00 | Argentina          |
+|  2724900.00 | Kazakstan          |
++-------------+--------------------+
+10 rows in set (0,00 sec)
+
+mysql> 
+
+
+
+5
+
+
+mysql> select Population,Name from city ORDER BY Population DESC LIMIT 10 ;
++------------+-------------------+
+| Population | Name              |
++------------+-------------------+
+|   10500000 | Mumbai (Bombay)   |
+|    9981619 | Seoul             |
+|    9968485 | São Paulo         |
+|    9696300 | Shanghai          |
+|    9604900 | Jakarta           |
+|    9269265 | Karachi           |
+|    8787958 | Istanbul          |
+|    8591309 | Ciudad de México  |
+|    8389200 | Moscow            |
+|    8008278 | New York          |
++------------+-------------------+
+10 rows in set (0,09 sec)
+
+
+6
+
+
+select Name,Population  from country where Name LIKE 'p%'  ORDER BY Population DESC LIMIT 3 ;
```

Comment:
Again, be careful - in this case it worked, but in other cases, there may be a difference between `'P%'` and `'p%'`.